include/envoy/server/filter_config.h

```diff
@@ -295,12 +295,23 @@ class NamedHttpFilterConfigFactory {
    */
   virtual ProtobufTypes::MessagePtr createEmptyConfigProto() { return nullptr; }
 
+  /**
+   * @return ProtobufTypes::MessagePtr create an empty virtual host, route, or weight cluster-local
+   *         config proto message for v2. The filter config, which arrives in an opaque
+   *         google.protobuf.Struct message, will be converted to JSON and then parsed into this
```

Comment:
Isn't the whole point of this change to hide `Struct` from the filters? If so, I would remove it from the comment as we might change to `Any` in the future as you pointed out. :)

Reply:
Done, good point :)